incubator/seq/README.md

```diff
@@ -0,0 +1,97 @@
+# Seq
+
+[Seq](https://getseq.net/) is the easiest way for development teams to capture, search and visualize structured log events! This page will walk you through the very quick setup process.
+
+## TL;DR;
+
+```bash
+$ helm install seq
+```
+
+## Introduction
+
+This chart bootstraps a [Seq](https://hub.docker.com/r/datalust/seq/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
We officially support the current and previous minor versions of Kubernetes. The chart won't work on 1.4 anyways.

Reply:
Fixed